applications/settings/desktop_settings/scenes/desktop_settings_scene_happy_mode.c

```diff
@@ -0,0 +1,59 @@
+#include <furi.h>
+#include <gui/scene_manager.h>
+#include <gui/view_dispatcher.h>
+#include <gui/modules/dialog_ex.h>
+#include <dolphin/dolphin.h>
+
+#include "desktop_settings_scene.h"
+#include "../desktop_settings_app.h"
+
+#define SCENE_EXIT_EVENT (0U)
```

Comment:
scene local events are bad, there are some places where they still left, but global enum is highly advised